tests/legacy/unit-tests/templates/functions.php

```diff
@@ -178,4 +178,16 @@ function( $gateways ) {
 
 		$this->assertEquals( $expected_html, $actual_html );
 	}
+
+	public function test_hidden_field() {
+		$actual_html = woocommerce_form_field('test',
```

Comment:
By default the `woocommerce_form_field` function prints to field to output. In order to make this work you'll need to add an argument for `return` set to `true`.